website/content/en/docs/olm-integration/generating-a-csv.md

```diff
@@ -1,208 +1,250 @@
+`operator-sdk generate kustomize manifests` generates a CSV kustomize base
+`config/manifests/bases/<operator-name>.clusterserviceversion.yaml` and a `config/manifests/bases/kustomization.yaml`
+by default. These files are required as `kustomize build` input in downstream commands.
 
-## Configuration
+By default, the command starts an interactive prompt if a CSV base in `config/manifests/bases` is not present
+to collect [UI metadata](#csv-fields). You can disable the interactive prompt by passing `--kustomize=false`.
 
-### Inputs
+```console
+$ operator-sdk generate kustomize manifests
+INFO[0000] Generating CSV manifest version 0.1.0
 
-The [ClusterServiceVersion (CSV)][doc-csv] generator requires certain input
-manifests to construct a CSV manifest. Each of these inputs are read every time
-`operator-sdk generate csv` is run are used to overwrite data in corresponding
-CSV fields (with one exception described [below](#csv-fields)).
+Display name for the operator (required):
+> memcached
 
-* Path to the Operator manifests root directory. By default `generate csv`
-extracts manifests from files in `deploy/` for the following kinds and adds
-them to the CSV. Use the `--deploy-dir` flag to change this path.
-    * Roles: `deploy/role.yaml`
-    * ClusterRoles: `deploy/cluster_role.yaml`
-    * Deployments: `deploy/operator.yaml`
-* Path to a directory containing CustomResourceDefinitions (CRDs) and Custom Resource examples (CRs).
-Use the `--crd-dir` flag to change this path.
-    * Custom Resources: `deploy/crds/<full group>_<version>_<kind>_cr.yaml`
-    * CustomResourceDefinitions: `deploy/crds/<full group>_<resource>_crd.yaml`
-* _For Go Operators only:_ Path to the API types root directory. The CSV generator
-also parses the [CSV markers][csv-markers] from the API type definitions
-to populate certain CSV fields. By default the API types directory is `pkg/apis/`.
-Use the `--apis-dir` flag to change this path. The CSV generator expects either
-of the following layouts for the API types directory.
-    * Mulitple groups: `<API-root-dir>/<group>/<version>/`
-    * Single groups: `<API-root-dir>/<version>/`
-* Interactive command prompts: `operator-sdk generate csv` provides the `--interactive`
-flag to optionally enable an interactive command prompt before generating a CSV
-that requests input for required and optional UI metadata fields. For example:
+Comma-separated list of keywords for your operator (required):
+> app, operator
+...
+```
 
-  ```console
-  $ operator-sdk generate csv  --csv-version 0.1.0 --interactive=true
-  INFO[0000] Generating CSV manifest version 0.1.0
+_For Go Operators only:_ the command parses [CSV markers][csv-markers] from Go API type definitions, located
```

Comment:
Nit: Maybe we can highlight this is a bit better like all our other caveats/notes.
```suggestion
**For Go Operators only:** the command parses [CSV markers][csv-markers] from Go API type definitions, located
```